Assets/MRTK/Providers/WindowsMixedReality/Shared/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "com.microsoft.mixedreality.toolkit.wmr",
     "displayName": "Mixed Reality Toolkit Windows Mixed Reality Provider Shared Components",
-    "version": "2.5.0-preview.0",
+    "version": "%version%",

```

Comment:
I hadn't thought of this before, but one downside to doing it this way (patching the version number at packaging time) is that these files now become invalid as-is and unusable for [referencing as a local package](https://docs.unity3d.com/Manual/upm-ui-local.htm). I think it would be good to support that.

Reply:
we support building local similarly to the way we do for nuget packages (run the script file). if you call createupmpackages.ps1 with "-OutputTarget local", they will be generated and copied to the value provided by -OutputDirectory

Reply:
works pretty well. i use it for testing changes :)